cli/src/pcluster/config/cluster_config.py

```diff
@@ -614,6 +615,26 @@ def existing_dns_name(self):
         ]["Nfs"]["DNSName"]
 
 
+class ExistingFsxFileCache(BaseSharedFsx):
+    """Represent the shared FSX FileCache resource."""
+
+    def __init__(self, file_cache_id: str, **kwargs):
+        super().__init__(**kwargs)
+        self.file_cache_id = file_cache_id
```

Comment:
Why are we are duplicating the information?

```
self.file_cache_id = file_cache_id
self.file_system_id = file_cache_id
```

Reply:
This was a workaround that we discussed, As the parent class uses file_system_id and I havent changed that class yet, was hoping to change that once I make changes in cookbook.

Reply:
Now I remember that we considered it out of scope for this specific PR to not make it bigger.
Let's go with your solution and fix it a a followup PR.
Thanks, approved.

Reply:
As discussed, will add resolution in a followup PR